config/jobs/ppc64le-cloud/build-docker/periodic-ci-docker.yaml

```diff
@@ -0,0 +1,166 @@
+            $PWD/upstream-master-ci/prow-info-docker.sh
+            rc=$?
+            [ $rc != 0 ] && echo "ERROR: prow-info-docker exited with code:$rc"
+            exit $rc
+          securityContext:
+            privileged: true
+          volumeMounts:
+          - name: boot
+            mountPath: /boot
+        volumes:
+        - name: boot
+          hostpath:
+            path: /boot/
+            
+    - name: periodic-build-dev-image-docker
+      cluster: k8s-ppc64le-cluster
+      cron: 0 3 * * *
+      decorate: true
+      decoration_config:
+        timeout: 2h
+      extra_refs:
+        - base_ref: main
+          org: ppc64le-cloud
+          repo: docker-ce-build
+      reporter_config:
+        slack:
+          channel: 'prow-job-notifications'
+          report_template: 'Job {{.Spec.Job}} of type {{.Spec.Type}} ended with state {{.Status.State}}. <!subteam^S02N6DWBX0F> <{{.Status.URL}}|View logs>'
+      spec:
+        containers:
+        - image: quay.io/powercloud/docker-ce-build@sha256:20890c8770ff57f91eb9f16fbb9f9e979958cd02b2a3c03795d2587e7b58a9db
+          resources:
+            requests:
+              cpu: "8000m"
+            limits:
+              cpu: "8000m"
+          command:
+          - /bin/bash
+          args:
+          - -c
+          - |
+            set -o errexit
+            set -o nounset
+            set -o pipefail
+            set -o xtrace
+            echo "* Start prow-build-dev-image *"
+            chmod ug+x $PWD/upstream-master-ci/prow-build-dev-image.sh
+            $PWD/upstream-master-ci/prow-build-dev-image.sh
+            rc=$?
+            [ $rc != 0 ] && echo "ERROR: prow-build-dev-image exited with code:$rc"
+            exit $rc
+          securityContext:
+            privileged: true
+            
+    - name: periodic-unit-test-docker
+      cluster: k8s-ppc64le-cluster
+      cron: 0 9 * * *
+      decorate: true
+      decoration_config:
+        timeout: 2h
+      extra_refs:
+        - base_ref: main
+          org: ppc64le-cloud
+          repo: docker-ce-build
+      reporter_config:
+        slack:
+          channel: 'prow-job-notifications'
+          report_template: 'Job {{.Spec.Job}} of type {{.Spec.Type}} ended with state {{.Status.State}}. <!subteam^S02N6DWBX0F> <{{.Status.URL}}|View logs>'
+      spec:
+        containers:
+        - image: quay.io/powercloud/docker-ce-build@sha256:20890c8770ff57f91eb9f16fbb9f9e979958cd02b2a3c03795d2587e7b58a9db
+          resources:
+            requests:
+              cpu: "8000m"
+            limits:
+              cpu: "8000m"
+          command:
+          - /bin/bash
+          args:
+          - -c
+          - |
+            set -o errexit
+            set -o nounset
+            set -o pipefail
+            set -o xtrace
+            echo "* Start prow-unit-test-docker *"
+            chmod ug+x $PWD/upstream-master-ci/prow-unit-test-docker.sh
+            $PWD/upstream-master-ci/prow-unit-test-docker.sh
```

Comment:
Hope junit xml file generation of results is taken care of in the script. Only then, we can expect a meaningful, easy to read test results in UI like in 
![image](https://github.com/ppc64le-cloud/test-infra/assets/63038004/b3e1cbf7-243d-4d4a-afe7-5de03c234299)

Reply:
We had a discussion about this and here's a summary:
We're running a large number of unit tests against github.com/moby/moby (3000+) with a negligible failure rate (a.t.m. a single test fails). Such a small number of failures are bound to happen and so we're really only concerned about the tests running and us being able to see the results somewhere, the UI logs contain all the information we need, we don't need the report. That said, we're going to work on getting it in the proper format in the near future. Also, just to add, the tests we're running are in [moby](github.com/moby/moby) and not in our repository anyway, we're just issuing a `make` command that runs these tests.